server/plugin.go

```diff
@@ -490,3 +547,46 @@ func addFields(fields []*model.SlackAttachmentField, title, msg string, short bo
 		Short: model.SlackCompatibleBool(short),
 	})
 }
+
+func messageToJSON(message string) ([]byte, error) {
+	messagefields := strings.Split(message, "\n")
+	if len(messagefields) > 0 {
```

Comment:
We can "return early" in this function like this

```suggestion
	if len(messagefields) == 0 {
	    return nil, errors.New("no message fields present in message string")
	}
```